products/BellHybrid/apps/application-bell-meditation-timer/presenter/MeditationTimerPresenter.cpp

```diff
@@ -14,12 +16,22 @@ namespace
     constexpr auto spinnerStep  = 1U;
     constexpr auto emptyValue   = 0U;
     constexpr auto defaultValue = 15U;
+    constexpr units::SOC lowBatteryThreshold{10};
+
+    bool isBatteryCharging(const Store::Battery::State state)
+    {
+        using State = Store::Battery::State;
+        return state == State::Charging or state == State::ChargingDone;
+    }
```

Comment:
```suggestion
    using State = Store::Battery::State;
    bool isBatteryCharging(State state)
    {
        return state == State::Charging or state == State::ChargingDone;
    }
```
The `const` is not necessary here. You can also move the `using` outside this function to shortcut the parameter type ;)

Reply:
done